### PR TITLE
[WIP]: test: next

### PR DESCRIPTION
**Pull request by koca**
exports everything from vtl
add tests for aspect ratio 

to be continued

### CODE REVIEW

File: packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js [hunk 1]
```diff
@@ -15,17 +14,53 @@ const renderComponent = (props) => {
   return render(base)
 }
 
+/**
```
Comment (koca) [inline packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js:17]: maybe i should add this to test-utils
but i'm not sure

File: packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js [hunk 2]
```diff
@@ -15,17 +14,53 @@ const renderComponent = (props) => {
...
+    for (j = 0; j < document.styleSheets[i].cssRules.length; ++j) {
+      if (sel.test(document.styleSheets[i].cssRules[j].selectorText)) {
+        // let selectorText = document.styleSheets[i].cssRules[j].selectorText
+        let cssText = document.styleSheets[i].cssRules[j].style.cssText
+        styles += cssText
+      }
+    }
+  }
+  return styles
+}
```
Comment (koca) [inline packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js:37]: maybe i should add `getElementStyles` to test-utils
but i'm not sure...

Comment (codebender828) [reply packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js:37]: Seems like a handy thing to have. I would also use this.

Comment (koca) [reply packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js]: moved

**Pull request merged**